docs/haskell.rst

```diff
@@ -89,14 +89,32 @@ special:
   contains a ``BUILD.bazel`` file is a *package*. You will learn about
   packages later in this tutorial.)
 
-To designate a directory as a Bazel workspace, create an empty file
+To designate a directory as a Bazel workspace, create a file
 named ``WORKSPACE`` in that directory.
+This file defines `external dependencies`_.
 
 When Bazel builds the project, all inputs and dependencies must be in
 the same workspace. Files residing in different workspaces are
 independent of one another unless linked, which is beyond the scope of
 this tutorial.
 
+Understand the WORKSPACE file
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+File ``tutorial/WORKSPACE`` defines how to obtain ``rules_haskell``
+and a compiler toolchain.  Because this tutorial lives in ``rules_haskell``'s
+repository, ``rules_haskell`` is made available as follows::
+
+  local_repository(
+      name = "rules_haskell",
+      path = "..",
+  )
+
+We refer to the template created by the start_ script
+to make ``rules_haskell`` available outside a local clone.
```

Comment:
```suggestion
If you want to set up the `WORKSPACE` file for your own repository,
run the start_ script and it will generate a fitting configuration.
```